environment-dev.yml

```
@@ -8,6 +8,7 @@ dependencies:
   - numpy
   - pooch >=1.8.0
   - pydantic >=2.0,<2.5.3  # FIXME: Remove pin once our dependencies (xclim, xscen) support pydantic 2.5.3
+  - spotpy
   - statsmodels
   - xarray
   - xclim >=0.45.0
```

Voir https://github.com/hydrologie/xhydro/pull/61#issuecomment-1904502979
```suggestion
  - xclim >=0.47.0
```